src/crimson/os/seastore/transaction_manager.h

```diff
@@ -311,6 +311,7 @@ class TransactionManager : public ExtentCallbackInterface {
       laddr_hint,
       len,
       ext->get_paddr(),
+      0,
```

Comment:
Worth a comment that the crc will be updated upon commit.

Reply:
Yeah, will do.